dev/deps/spark-deps-hadoop-2.2

```diff
@@ -6,6 +6,7 @@ antlr-2.7.7.jar
 antlr-runtime-3.4.jar
 antlr4-runtime-4.5.2-1.jar
 aopalliance-1.0.jar
+aopalliance-repackaged-2.4.0-b34.jar
```

Comment:
We'll probably have to look at the impact on the LICENSE file for all these dependency changes. I can help do that bit if once we're pretty sure this will work.